packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -271,7 +271,7 @@ export function signInActor({ services }: SignInMachineOptions) {
           exit: ['clearFormValues', 'clearError', 'clearTouched'],
           states: {
             edit: {
-              entry: sendUpdate(),
+              entry: 'sendUpdate',
```

Comment:
I'm very new to xstate and do not know the expected patterns so please bear with me here, but would it make sense to use a string `enum` for the `entry` values in this file? Probably out of scope either way 😄

Reply:
Nope, unfortunately it's the other way around. We pass non-enum'd string as the key here, and then xstate will use that key to infer the types. Technically, you can still pass strictly typed string here since it `extends string` anyway, but I don't believe that's an established pattern.

But that said, xstate will warn us if there are string typos through its [typegen](https://xstate.js.org/docs/guides/typescript.html#typegen) or through its [vscode extension](https://marketplace.visualstudio.com/items?itemName=statelyai.stately-vscode).